benchmarks/CMakeLists.txt

```diff
@@ -1,37 +1,49 @@
 cmake_minimum_required(VERSION 2.8.3)
-project(moveit_ros_benchmarks)
+project(moveit_benchmarks)
```

Comment:
Is this going to be a general pattern going forward of removing the *_ros_* specifier to our packages? I would like that, but it seems odd that we would only do this package.